pkg/storage/secret/metadata/secure_value_model.go

```diff
@@ -237,7 +236,7 @@ func toRow(sv *secretv1beta1.SecureValue, externalID string) (*secureValueDB, er
 		Version: sv.Status.Version,
 
 		Description: sv.Spec.Description,
-		Keeper:      toNullString(sv.Spec.Keeper),
+		Keeper:      toNullString(&keeper),
```

Comment:
Does this mean that we will end up with rows that are both `null` (prior to this change) and those which are `system` ? Don't know if there's a way to add a `not null default 'system'` constraint migration, if that makes sense?